api/src/public/login/script.js

```diff
@@ -28,7 +28,11 @@ var handleResponse = function(xmlhttp) {
   if (xmlhttp.status < 200) {
     setState('loginoffline');
   } else if (xmlhttp.status === 200) {
-    window.location = unescape(document.getElementById('redirect').value);
+    if (JSON.parse(xmlhttp.response).canCongifure) {
+      window.location = JSON.parse(xmlhttp.response).redirect;
+    } else {
+      window.location = unescape(document.getElementById('redirect').value);
+    }
```

Comment:
Ideally we'd have one way of getting the URL not two. Can we remove the `redirect` element from the HTML and always return the path in the response? That way you can get rid of the `canConfigure` property.

If you wanted to do this semantically you could return a redirect response from API, something like:

```js
res.redirect(307, path.join(...)); // 307: Temporary Redirect tells the browser not to cache it
```

Then in the script.js, do something like...

```js
...
} else if (xmlhttp.status === 307) {
  window.location = response.redirectUrl;
} else {
...
```